sdk/monitor/monitor-query/test/setupOpenTelemetry.ts

```diff
@@ -1,44 +1,45 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AzureMonitorTraceExporter } from "@azure/monitor-opentelemetry-exporter";
-import { NodeTracerProvider } from "@opentelemetry/node";
-import { BatchSpanProcessor } from "@opentelemetry/tracing";
-import { TracerProvider } from "@opentelemetry/api";
-import * as dotenv from "dotenv";
-import { Context } from "mocha";
-import { getAppInsightsConnectionString } from "./public/shared/testShared";
-
-dotenv.config();
-
-export function runWithTelemetry(
-  mochaContext: Pick<Context, "skip">,
-  fn: (provider: TracerProvider) => void
-): Promise<void> {
-  const appInsightsConnectionString = getAppInsightsConnectionString(mochaContext);
-
-  const provider = new NodeTracerProvider({
-    // plugins: {
-    //   https: {
-    //     // Ignore Application Insights Ingestion Server
-    //     ignoreOutgoingUrls: [new RegExp(/dc.services.visualstudio.com/i)]
-    //   }
-    // }
-  });
-  provider.register();
-
-  // Create an exporter instance
-  const exporter = new AzureMonitorTraceExporter({
-    connectionString: appInsightsConnectionString
-  });
-
-  const batchSpanProcessor = new BatchSpanProcessor(exporter);
-
-  // Add the exporter to the provider
-  // TODO: there is an API incompatiblity here.
-  provider.addSpanProcessor(batchSpanProcessor as any);
-
-  fn(provider);
-
-  return provider.shutdown();
-}
+// import { AzureMonitorTraceExporter } from "@azure/monitor-opentelemetry-exporter";
```

Comment:
why commented?

Reply:
I have disabled the tests using opentelemetry-exporter for data ingestion - bcz there is a bug in that package - also bcz it doesn't support browser. We don't really need tests to do the data ingestion. So commenting it out for now.